enterprise/admin/views.py

```diff
@@ -291,6 +291,13 @@ def _handle_bulk_upload(cls, enterprise_customer, manage_learners_form, request,
 
         return all_processable_emails
 
+    # TODO: this method is HUGE (> 100 lines) and complicated. Refactor it.
+    # Potential improvements:
+    # 1. Restructure flow so that notifications are sent after users and pending users are enrolled
+    #    FIXME: `send_email_notification_message` for pending users are executed OUTSIDE of ConditionalEmailConnection
```

Comment:
As with the TODO, when do we plan on addressing this FIXME?  If it will not be a part of this PR we should log a follow-up ticket so we don't forget about it.